server/rpc_server.go

```diff
@@ -255,6 +255,13 @@ func MonitorRPCRequest() func(ctx context.Context, methodName string, err error)
 	start := time.Now()
 	return func(ctx context.Context, methodName string, err error) {
 		m := rpcEndpointMetrics["rpc."+methodName]
+		if m==nil{
+			registeredRPCEndpoints := make([]string, 0, len(rpcEndpointMetrics)) //for a better error-message retrieve all correctly registered rpcEndpoints
+			for registeredRPCEndpoint := range rpcEndpointMetrics {
+				registeredRPCEndpoints = append(registeredRPCEndpoints, registeredRPCEndpoint[4:len(registeredRPCEndpoint)]) //to hide internals slice away the "rpc."-part
+			}
+			fmt.Printf("rpc server panic: the methodName '%v' listed in the second parameter of MonitorRPCRequest() was not registered with the server.\nRegistered methodNames are: %v. Please check for spelling mistakes.\n\n", methodName, registeredRPCEndpoints)
```

Comment:
I'm thinking this is a little more verbose than we need. Let's just cut it down to a message like `unable to monitor rpc request. unknown method name: %s` and wrap it with a `Log.Errorf` instead of a plain print statement, which will alert the user but not kill the server.

That should at least be enough tip the user into looking at the list of registered methods.